core/translate/group_by.rs

```diff
@@ -363,6 +378,11 @@ pub fn emit_group_by<'a>(
         });
     }
 
+    let filtered_results = plan
+        .result_columns
+        .iter()
+        .filter(|rc| !rc.contains_aggregates)
+        .collect::<Vec<_>>();
```

Comment:
This should be called `non_aggregate_result_columns` and I don't think we need to `.collect()`